src/components/LoginPage/FacebookLogin/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./index.css";
+import FacebookLoginWithButton from "react-facebook-login";
+
+const componentClicked = () => {
+  console.log("Clicked!");
+};
+
+const LoginButton = ({ facebookResponse }) => (
+  <FacebookLoginWithButton
+    appId="2667757206808803"
+    // autoLoad
+    fields="name,email,picture"
+    onClick={componentClicked}
+    callback={facebookResponse}
+    icon="fa-facebook"
+  />
+);
+
+const UserScreen = ({ user }) => (
+  <>
+    <h1>Welcome {user.name}!</h1>
+    <p>{user.email}</p>
+    <img
+      src={user.picture.data.url}
+      height={user.picture.height}
+      width={user.picture.width}
+      alt="avatar"
+    />
+  </>
+);
+
+class Login extends React.Component {
+  state = { user: false };
+  facebookResponse = (response) => {
+    console.log(response);
+    this.setState({ ...this.state, user: response });
+  };
+
+  render() {
+    return (
+      <div style={{ margin: "auto", textAlign: "center", width: "13em" }}>
```

Comment:
Please add the style in a separate css file

Reply:
PTAL